packages/turf-unkink-polygon/test.js

```diff
@@ -36,6 +36,28 @@ test("unkink-polygon", (t) => {
   t.end();
 });
 
+test("issue #2504", (t) => {
+  // fill coords with a circle with an arbitrary number of points
+  const coords = [];
+  const points = 1000000;
```

Comment:
This number has to be *really* large in node, I think the limit in Chrome is somewhat smaller